File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem56_KokoEatingBananasTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class Problem56_KokoEatingBananasTest {

    private final Problem56_KokoEatingBananas test = new Problem56_KokoEatingBananas();

    @Test
    void minEatingSpeed() {
        int[] piles = new int[]{3, 6, 7, 11};

        assertEquals(4, test.minEatingSpeed(piles, 8));
    }
}

File: JavaScript/leetcode75/src/test/Problem56_KokoEatingBananas.test.js
const minEatingSpeed = require("../main/Problem56_KokoEatingBananas");

test("should return minimum eating speed", () => {
  const piles = [3, 6, 7, 11],
    h = 8;

  expect(minEatingSpeed(piles, h)).toBe(4);
});
